source/index.js

```diff
@@ -654,10 +645,25 @@ function normalizeArguments(url, options) {
 	return options;
 }
 
+function assignOptions(defaults, options = {}) {
```

Comment:
The `= {}` is no longer needed here.

Reply:
Nope. Tests have failed :laughing:

Reply:
Ugh, yeah I didn't notice `options` was used outside the `extend` call...